Druid-Tests/DRProductionBytecodeTest.class.st

```diff
@@ -219,10 +217,13 @@ DRProductionBytecodeTest >> testBytecodePushLiteralConstant [
 	self
 		executePrimitiveWithReceiver: (memory integerObjectOf: 17)
 		withArguments: #(  ).
-		
+
 	self
 		assert: machineSimulator receiverRegisterValue
-		equals: (memory integerObjectOf: 42)
+		equals: (memory integerObjectOf: 42).
+
+	self assert:
+		(cogit hasAnnotatedAbstractInstructions: cogit getIsObjectReference)
```

Comment:
Yes, I couldn't make that test (bah, it will take me more time and I wanted to know if I was by the right way).
That test is more integral, I think complementary to this one.